src/applications/coronavirus-vaccination-expansion/containers/IntroductionPage.jsx

```diff
@@ -49,7 +48,12 @@ class IntroductionPage extends React.Component {
       'button-click-label': 'I have used VA health care before',
       'button-background-color': '#0071bb',
     });
-    window.location.href = this.state.nextUrl;
+
+    if (this.state.nextUrl === alreadyReceivingCarePath) {
+      document.location.assign(alreadyReceivingCarePath);
```

Comment:
`alreadyReceivingCarePath` is a URL that is not under this application's root URL so we essentially do a full redirect to that other URL

Reply:
That sounds right. If you are already receiving care, you are bounced out of `coronavirus-vaccination-expansion` and over to the existing `coronavirus-vaccination` (KMI) app.